file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -129,84 +134,95 @@ protected internal override async Task OnExecuteCommand()
             IsCanceled = false;
             CancellationTokenSource = new CancellationTokenSource();
             CancellationToken cancellationToken = CancellationTokenSource.Token;
-            
+
             try
             {
-                logger.LogInformation("Execute job started for Action : {Action} and displayName :{displayName} .",Action,DisplayName);
+                logger.LogInformation("Execute job started for Action : {Action} and displayName :{displayName} .", Action, DisplayName);
                 var fileShareClient = fileShareClientFactory();
-                var buildBatchModel = BuildBatchModel();                
-                logger.LogInformation("File Share Service batch create started.");
-                batchHandle = await fileShareClient.CreateBatchAsync(buildBatchModel, CancellationToken.None); 
-                logger.LogInformation("File Share Service batch create completed for batch ID:{BatchId}.", batchHandle.BatchId);
-                FileUploadProgress.Clear();
-                try
+                var buildBatchModel = BuildBatchModel();
+                BatchSearchResponse batchSearchResponse = await SearchBatch();
```

Comment:
Could you please ensure any exceptions raised by `SearchBatch()` are handled gracefully. At the moment an exception from `SearchBatch()` is caught by the global exception handler.

Reply:
added try-catch block. Thanks